File: mobile/ios/CortexV2-Emotiv-iOS/CortexV2-Emotiv-ObjC/CortexV2-Emotiv-ObjC/HeadsetTableViewCell.m
//
//  HeadsetTableViewCell.m
//  CortexV2-Emotiv-ObjC
//
//  Created by Viet Anh on 3/18/20.
//  Copyright © 2020 Emotiv. All rights reserved.
//

#import "HeadsetTableViewCell.h"

@implementation HeadsetTableViewCell

- (void)awakeFromNib {
    [super awakeFromNib];
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

+ (NSString *)reuseIdentifier {
    return @"CustomCellIdentifier";
}


@end

File: mobile/ios/CortexV2-Emotiv-iOS/CortexV2-Example-Swift/CortexV2-Example-Swift/HeadsetTableViewCell.swift
//
//  HeadsetTableViewCell.swift
//  CortexV2-Example-Swift
//
//  Created by Viet Anh on 3/16/20.
//  Copyright © 2020 Emotiv. All rights reserved.
//

import UIKit

class HeadsetTableViewCell: UITableViewCell {
    @IBOutlet weak var headsetName: UILabel!
    @IBOutlet weak var connectedBy: UILabel!
    @IBOutlet weak var status: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }
    
}
